app/Imports/applicantImport.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Imports;
+
+use Carbon\Carbon;
+use Modules\CodeTrek\Entities\CodeTrekApplicant;
+use Illuminate\Support\Collection;
+use Maatwebsite\Excel\Concerns\ToCollection;
+use Maatwebsite\Excel\Concerns\WithHeadingRow;
+
+class ApplicantImport implements ToCollection, WithHeadingRow
```

Comment:
Please update the all places. And please change the file name accordingly and use TitleCase for the file name.
```suggestion
class CodeTrekApplicantImport implements ToCollection, WithHeadingRow
```